independent-projects/qute/core/src/main/java/io/quarkus/qute/TemplateLocator.java

```diff
@@ -5,6 +5,8 @@
 
 /**
  * Locates template sources. The locator with higher priority takes precedence.
+ * Quarkus automatically register all CDI beans that implement the locator
```

Comment:
Quarkus automatically register**s**... that implement **this interface** with...

Reply:
Fixed, thank you